.github/workflows/VeniceCI-E2ETests.yml

```diff
@@ -64,14 +68,28 @@ jobs:
         env:
          NODE_OPTIONS: --max-old-space-size=9182
         with:
-         token: ${{ secrets.GITHUB_TOKEN }}
-         name: ${{ github.job }} Test Reports       # Name where it report the test results
-         path: '**/TEST-*.xml'
-         fail-on-error: 'false'
-         max-annotations: '10'
-         list-tests: 'all'
-         list-suites: 'all'
-         reporter: java-junit
+          token: ${{ secrets.GITHUB_TOKEN }}
+          name: ${{ github.job }} Test Reports       # Name where it report the test results
+          path: '**/TEST-*.xml'
+          fail-on-error: 'false'
+          max-annotations: '10'
+          list-tests: 'all'
+          list-suites: 'all'
+          reporter: java-junit
+      - name: Publish Test Report
+        continue-on-error: true
+        env:
+          NODE_OPTIONS: "--max_old_space_size=8192"
```

Comment:
Why do we need 8G on publish test report though

Reply:
This ensures the plugin functions correctly by increasing the Node.js heap size. Previously, report generation would fail due to running out of memory, so we set NODE_OPTIONS: "--max_old_space_size=8192" to resolve the issue.